src/styles/style.css

```diff
@@ -195,6 +195,11 @@ ol {
   list-style-image: none;
 }
 
+table, th, td {
+  border: 1px solid #d1dce5;
+  padding: 20px;
+}
+
```

Comment:
Right, or the table could be put inside a container with `overflow-y: auto` so that when the table overflows, it could be scrolled horizontally and it would not affect the entire page. Though if scrolling could be prevented somehow by design change, it would be better